docs/api/en/helpers/ArrowHelper.html

```diff
@@ -46,8 +46,7 @@ <h3>[name]([param:Vector3 dir], [param:Vector3 origin], [param:Number length], [
 		[page:Vector3 origin] -- Point at which the arrow starts.<br />
 		[page:Number length] -- length of the arrow. Default is *1*.<br />
 		[page:Number hex] -- hexadecimal value to define color. Default is 0xffff00.<br />
-		[page:Number headLength] -- The length of the head of the arrow. Default is 0.2 * length.<br />
```

Comment:
Wait, something is wrong. Did you mean to delete this line?

Reply:
Oops. So sorry. Can you please restore that line?

Reply:
Oops. Sorry. I will fix it.

Reply:
Done!

Reply:
Thanks!

Reply:
Also, I think this is what you intended to do? https://github.com/mrdoob/three.js/commit/ff61617519b5fac9925538e9c6f2922e2e6690e7

Reply:
I opted to leave the original until the new text was translated. But your version is OK, too.